### PR TITLE
add rake task to fix orphan records

### PR DESCRIPTION
**Pull request by isaacmg410**
This PR adds a rake task that assign a ParticipatoryProcess non published to records that don't have component or proces in order to fix them and don't crash in a 0.15 migration.

### CODE REVIEW

**Comment by isaacmg410** [comment]
@tramuntanal or @aitorlb can you review it, please?

File: lib/tasks/participa.rake [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
```
Comment (tramuntanal) [inline lib/tasks/participa.rake]: Can you pleas add a description of what are this rake tasks about: explain the problem in the header of the file; and the solutions that each tasks offer in each task's description.

Comment (isaacmg410) [reply lib/tasks/participa.rake]: Done!

File: lib/tasks/participa.rake [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+# there may be problems with some migrations, and they will fail.
+namespace :participa do
+  # Components ------
+  #
+  # Find the Organization.
+  # Find or Create an unpublished participatory process
+  # Takes all the components that don't have a participatory space related.
+  # Assign the created participatory process to the components
+  # without participatory space .
+  desc 'Fix Orphan Component records'
```
Comment (tramuntanal) [inline lib/tasks/participa.rake]:
```suggestion
  desc <<EODESC
Fix Orphan Component records:
- Find the Organization.
- Find or Create an unpublished participatory process
- Takes all the components that don't have a participatory space related.
- Assign the created participatory process to the components without participatory space.
EODESC
```

Comment (isaacmg410) [reply lib/tasks/participa.rake]: I have followed the same description system as other rake tasks in Decidim.
As for example thsse one:
 https://github.com/decidim/decidim/blob/master/decidim-core/lib/tasks/decidim_metrics_tasks.rake 
or this https://github.com/decidim/decidim/blob/master/decidim-core/lib/tasks/decidim_data_portability_tasks.rake

If you prefer to use the own rake's system, I can change it.
What you prefer?

**Pull request merged**